### PR TITLE
{lib}[foss/2018a] TensorFlow v1.7.0 w/ Python 3.6.4 + CUDA 9.1.85

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.7.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+builddependencies = [
+    ('Bazel', '0.11.1'),
+    ('wheel', '0.30.0', '-Python-%(pyver)s'),
+]
+dependencies = [
+    ('Python', '3.6.4'),
+    ('cuDNN', '7.0.5', '-CUDA-%s' % cuda_ver, True),
+]
+
+with_mkl_dnn = False
```
Comment (boegel) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.7.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb]: @smoors Why is this required? Maybe clarify this with a comment above this line?

**Comment by boegel** [comment]
@smoors Please submit a test report for this?

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.7.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.7.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb]: don't know, I just copied from the previous version. maybe @wpoely86 knows more?

**Comment by boegel** [comment]
@smoors Any reason for not using TF 1.8?

**Comment by smoors** [comment]
no I made the easyconfig before 1.8 came out :)

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.7.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb (hunk 1 above)
Comment (wpoely86) [reply easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.7.0-foss-2018a-Python-3.6.4-CUDA-9.1.85.eb]: it can be dropped. The easyblock should handle it.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic167 - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/7f5033ef06ed9b8ceb4e55a8a09ed612 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!
